src/ui/sass/modules/_Pagination.scss

```diff
@@ -55,10 +55,28 @@ $pagination-color-hover: var(--yxt-color-text-secondary) !default;
     }
   }
 
-  #active-page {
+  &-active {
     background-color: var(--yxt-pagination-color-active-page);
   }
 
+  &-active-onepage-desktop {
```

Comment:
Aligned with the change below to add more classes, could we only add this `background-color: inherit` for the classes that need them?

e.g.

```
&-active {
  &--onePageMobile {
    @media (max-width: $breakpoint-mobile-max) {
      background-color: inherit;
    }
  }
}
```

Reply:
As an afterthought, perhaps we now have a different meaning for `active`. Instead of adding this reset styling to remove the background-color, maybe we can start passing in classes to give active styling. That is, instead of always having the styling on line 59, perhaps we can pass in activeDesktop or activeMobile with corresponding classes to add background-color with their respective media breakpoints. What do you think?